source/kernel/src/drivers/emb_display.rs

```diff
@@ -0,0 +1,260 @@
+use crate::{
+    comms::{
+        kchannel::{KChannel, KConsumer},
+        oneshot::Reusable,
+    },
+    registry::{
+        Envelope, KernelHandle, Message, RegisteredDriver, ReplyTo,
+    },
+    Kernel,
+};
+use maitake::sync::Mutex;
+use mnemos_alloc::containers::{HeapArc, HeapArray, HeapFixedVec};
+// use tracing::{debug, warn};
+use uuid::Uuid;
+use embedded_graphics::{
+    pixelcolor::{Gray8, GrayColor},
+    prelude::*,
+    image::ImageRaw,
+};
+
+const BYTES_PER_PIXEL: u32 = 1;
+const DISP_WIDTH: u32 = 319;
+const DISP_HEIGHT: u32 = 239;
+
+// Registered driver
+pub struct EmbDisplay {
+    kernel: &'static Kernel,
+}
+
+// FrameChunk is recieved after client has sent a request for one
```

Comment:
nit: is this supposed to be a doc comment?
```suggestion
/// FrameChunk is recieved after client has sent a request for one
```

Reply:
Also just a standard comment. I should probably put in document comments, come to think of it